### PR TITLE
vip components

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Downloaded built and published versions of VIP widget components/grapesjs to publish under the `saasquatch` npm org

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/vip-vanilla-components-grapesjs/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+```sh
+$ npm start
+```
+
+
+
+
+
+## License
```
Comment (DerekSiemens) [inline packages/vip-vanilla-components-grapesjs/README.md:118]: I don't think we want this BSD 3-clause here

**Comment by DerekSiemens** [comment]
Read me's could definitely use some work but overall looks good

File: packages/vacay-vip-components/package.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@saasquatch/vacay-vip-components",
+  "version": "1.0.0-0",
```
Comment (DerekSiemens) [inline packages/vacay-vip-components/package.json:3]: Flagging pre-release

File: packages/vip-vanilla-components/package.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@saasquatch/vip-vanilla-components",
+  "version": "1.0.0-4",
```
Comment (DerekSiemens) [inline packages/vip-vanilla-components/package.json:3]: Flagging pre-release

**Comment by DerekSiemens** [comment]
Looking good, good to continue with the publishing and tagging then we can merge the PR

**Comment by 00salmon** [comment]
all published ✅

**Review by DerekSiemens — APPROVED** [review]

**Pull request merged**